### PR TITLE
docs: update multivariate anomaly detection - fabric doc edit

### PR DESCRIPTION
**Pull request by JessicaXYWang**

### CODE REVIEW

**Review by mhamilton723 — DISMISSED** [review]

File: docs/Explore Algorithms/OpenAI/Quickstart - Understand and Search Forms.ipynb [hunk 1]
```diff
@@ -266,7 +266,7 @@
    "source": [
     "## 4 - Simplify form recognition output\n",
     "\n",
-    "This code uses the [FormOntologyLearner](https://mmlspark.blob.core.windows.net/docs/0.10.0/pyspark/synapse.ml.cognitive.html#module-synapse.ml.services.form.FormOntologyTransformer), a transformer that analyzes the output of Form Recognizer transformers and infers a tabular data structure. The output of AnalyzeInvoices is dynamic and varies based on the features detected in your content.\n",
+    "This code uses the [FormOntologyLearner](https://mmlspark.blob.core.windows.net/docs/0.10.0/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.FormOntologyTransformer), a transformer that analyzes the output of Form Recognizer transformers (for Azure AI Document Intelligence) and infers a tabular data structure. The output of AnalyzeInvoices is dynamic and varies based on the features detected in your content.\n",
```
Comment (mhamilton723) [inline docs/Explore Algorithms/OpenAI/Quickstart - Understand and Search Forms.ipynb:269]: This link 

(https://mmlspark.blob.core.windows.net/docs/0.10.0/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.FormOntologyTransformer

is very old, please update to new one

File: docs/Get Started/Quickstart - Your First Models.ipynb [hunk 2]
```diff
@@ -11,8 +11,8 @@
     }
    },
    "source": [
-    "# Build your first SynapseML models\n",
-    "This tutorial provides a brief introduction to SynapseML. In particular, we use SynapseML to create two different pipelines for sentiment analysis. The first pipeline combines a text featurization stage with LightGBM regression to predict ratings based on review text from a dataset containing book reviews from Amazon. The second pipeline shows how to use prebuilt models through the Azure AI Services to solve this problem without training data."
+    "# Build your first SynapseML model\n",
+    "This article introduces how to build your first machine learning model using SynapseML and demonstrates how SynapseML simplifies complex machine learning tasks. We use SynapseML to create a small ML training pipeline that includes a featurization stage and a LightGBM regression stage. The pipeline predicts ratings based on review text from a dataset of Amazon book reviews. Finally, we showcase how SynapseML simplifies the use of prebuilt models to solve ML problems.\n"
```
Comment (mhamilton723) [inline docs/Get Started/Quickstart - Your First Models.ipynb:15]: Not sure this is that much of an improvement

File: docs/Explore Algorithms/OpenAI/OpenAI.ipynb [hunk 3]
```diff
@@ -217,9 +226,9 @@
     }
    },
    "source": [
-    "## Step 6: Transform the dataframe with the OpenAICompletion Client\n",
+    "## Transform the dataframe with the OpenAICompletion Client\n",
     "\n",
-    "Now that you have the dataframe and the completion client, you can transform your input dataset and add a column called `completions` with all of the information the service adds. We'll select out just the text for simplicity."
+    "After completing dataframe and the completion client, you can transform your input dataset and add a column called `completions` with all of the information the service adds. Select just the text for simplicity."
```
Comment (mhamilton723) [inline docs/Explore Algorithms/OpenAI/OpenAI.ipynb:231]:
```suggestion
    "After creating the dataframe and the completion client, you can transform your input dataset and add a column called `completions` with all of the information the service adds. Select just the text for simplicity."
```

**Review by mhamilton723 — COMMENTED** [review]
Two small nits! Other than that lovely

File: docs/Explore Algorithms/OpenAI/Quickstart - Understand and Search Forms.ipynb (hunk 1 above)
Comment (JessicaXYWang) [reply docs/Explore Algorithms/OpenAI/Quickstart - Understand and Search Forms.ipynb]: update to 1.0.4

File: docs/Get Started/Quickstart - Your First Models.ipynb (hunk 2 above)
Comment (JessicaXYWang) [reply docs/Get Started/Quickstart - Your First Models.ipynb]: using the original version

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**